### ISSUE TITLE
Recovery is repeatedly showing error (Money stuck)

### ISSUE THREAD

**Issue by moneky345** [polymorpher/one-wallet#309]
Hi,

I'm very disappointed as there are some incredibly low limits and the function to increase the does not even work.

I sent my money there not knowing about limits.

I need the limits removed completely. How do do this?

**Comment by polymorpher** [comment]
Set recover address and get all money out. Try creating another wallet and read the text next time

**Comment by moneky345** [comment]
Thanks for instant response. That's impressive.

Yes, that was my mistake, I just did not expect any limits in a defi wallet.

Now could you please help me with step by step instructions?

**Comment by polymorpher** [comment]
It's here

<img width="760" alt="image" src="https://user-images.githubusercontent.com/913871/165253099-52b21fb3-3258-4456-b06d-70e2e38017cb.png">

**Comment by moneky345** [comment]
Thanks. I did sent the recovery address but it keeps failing

![image](https://user-images.githubusercontent.com/104420960/165253893-9c5cd22a-8609-4457-bf55-5516616596d3.png)

**Comment by polymorpher** [comment]
sorry - that's a bug caused by recent updates, I will fix later tonight.

**Comment by moneky345** [comment]
Thank you! Could you please ping me here once this is done?

**Comment by polymorpher** [comment]
Fixed now. Sorry, was too busy with another issue last night

(https://github.com/polymorpher/one-wallet/commit/9c14f29268da96f682ff59647f6ad0fa6f9aac11)

**Comment by moneky345** [comment]
Thanks, however, I still seem to be getting errors.



![image](https://user-images.githubusercontent.com/104420960/165380006-3f060c48-dceb-473e-8085-9ad45c2dfb78.png)

**Comment by polymorpher** [comment]
Did you refresh the page?

**Comment by moneky345** [comment]
Worked after refreshing! Thank you, you are a life saver :)
